README.md

```diff
@@ -391,6 +391,13 @@ export interface JestPrismaEnvironmentOptions {
    */
   readonly verboseQuery?: boolean;
 
+  /**
+   *
+   * Display SQL queries with parameters in test cases to STDOUT.
+   *
+   */
+  readonly verboseQueryWithPrameters?: boolean;
```

Comment:
I think it's better to change `verboseQuery` behavior  rather than to add new option .

Reply:
@Quramy 
thanks, I have made the suggested changes!
[671dd45](https://github.com/Quramy/jest-prisma/pull/99/commits/671dd4558f7e0638335d5761a5f72d14b6646aea)